### ISSUE TITLE
Using this on MT 5.5.0 throws error, crashes to main menu

### ISSUE THREAD

**Issue by Baigle** [random-geek/chatcolor#1]
```
ModError: Failed to load and run mod "chatcolor":
chatcolor:init.lua:158: attempt to call field 'register_on_connect' (a nil value)
stack traceback:
	chatcolor:init.lua:158: in main chunk
Check debug.txt for details.
```

Crashes on both singleplayer and multiplayer servers

**Comment by random-geek** [comment]
Thanks for bringing this up. This mod probably needs some maintenance anyway, so I'll take a look at it.

**Comment by random-geek** [comment]
Fixed by b8cc61843b102f96406ed21e704300c171adb5af. I also did some "maintenance," e.g. rewrote the entire mod. There's a few more things I want to change, but hopefully it works at the moment.

I saw you have a fork as well; sorry for changing everything!
